engine/runtime-integration-tests/src/test/java/org/enso/interpreter/node/typecheck/TypeCheckValueTest.java

```diff
@@ -0,0 +1,70 @@
+package org.enso.interpreter.node.typecheck;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.oracle.truffle.api.CallTarget;
+import org.enso.interpreter.runtime.data.EnsoMultiValue;
+import org.enso.interpreter.runtime.data.Type;
+import org.enso.test.utils.ContextUtils;
+import org.enso.test.utils.TestRootNode;
+import org.graalvm.polyglot.Context;
+import org.junit.Test;
+
+public class TypeCheckValueTest {
+  private static Context ctx;
+
+  private static Context ctx() {
```

Comment:
minor: Until we resolve https://github.com/enso-org/enso/issues/11683, ensure that you dispose context in the test, similarly to how it is done in [TypeOfNodeMultiValueTest](https://github.com/enso-org/enso/blob/c780c37fa23e850cb63eb2b4b5200af0d6fd96e6/engine/runtime-integration-tests/src/test/java/org/enso/interpreter/node/expression/builtin/meta/TypeOfNodeMultiValueTest.java#L110-L116)

Reply:
Thank you. Closing since d3102adcd6